hack/ensure-go.sh

```diff
@@ -31,7 +31,7 @@ EOF
   local go_version
   IFS=" " read -ra go_version <<< "$(go version)"
   local minimum_go_version
-  minimum_go_version=go1.12.1
+  minimum_go_version=go1.13.4
```

Comment:
Do we specifically need to ensure minimum is 1.13.4 here? or can we just ensure it is at least 1.13.0 for the local compiler?

Reply:
Makes sense yeah